.gitignore

```diff
@@ -2,3 +2,19 @@
 /node_modules
 /dist
 /coverage
+
+# IDEs and editors
```

Comment:
Things like this should be in your `/home/<user>/.gitignore`

Reply:
Removed this changes